### ISSUE TITLE
False positive for `no-commented-out-code` with `// #endregion` in class

### ISSUE THREAD

**Issue by Arttse** [cartant/eslint-plugin-etc#29]
For example:
```ts
class FooBar {
  // #region Some region
  one() {}

  two() {}

  three() {}
  // #endregion
}
```
triggered for `// #endregion`

**Comment by cartant** [comment]
Should be fixed in `1.3.8`.
